Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```
@@ -591,26 +591,36 @@
   - type: Clothing
-    sprite: Clothing/Belt/syndieholster.rsi
+    sprite: Clothing/Neck/Misc/syndieholster.rsi
+    equipSound:
+      path: /Audio/SimpleStation14/Items/Equip/toolbelt_equip.ogg
   - type: Item
     size: Ginormous
   - type: Storage
     maxItemSize: Huge
     grid:
-    - 0,0,3,3
+    - 0,0,3,4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)